test/parallel/test-process-resource-usage.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+require('../common');
+const assert = require('assert');
+const usage = process.resourceUsage();
+
+assert(usage.userCpuTimeUsedSec >= 0);
```

Comment:
True, even I was worried about that. At least, we know for sure that except for few of them, others will be zero in Windows (according to the documentation). We can assert that, right?